File: app/controllers/Standings.java
package controllers;

import play.*;
import play.mvc.*;
import views.html.*;
import models.*;

public class Standings extends Controller {

	//Standing home page (returns template/html)
	public static Result index() {
		return ok(standings.render("Standings Home Page"));
	}
	
	//getAll (returns JSON)
	public static Result getAll() {
		return ok(StandingsManager.getAllStandingsJSON());
	}
}

File: app/controllers/Standings.scala
package controllers

import play.api._
import play.api.mvc._
import play.api.libs.json._
import models._
import views._


object Standings extends Controller {
	//Standing home page (returns template/html)
	def index = Action {
		Ok(html.standings("Standings Home Page"))
	}

	//getAll (returns JSON)
	def getAll = Action {
		Ok(play.api.libs.json.Json.parse(StandingsManager.getAllStandingsJSON().toString()))
	}
}
